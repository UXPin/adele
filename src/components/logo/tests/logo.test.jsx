import { mount } from 'enzyme';
import React from 'react';
import 'jest-styled-components';
import { BrowserRouter } from 'react-router-dom';

import Logo from '../logo';

it('renders Logo correctly', () => {
  const wrapper = mount(<BrowserRouter><Logo /></BrowserRouter>);
  expect(wrapper).toMatchSnapshot();
});
