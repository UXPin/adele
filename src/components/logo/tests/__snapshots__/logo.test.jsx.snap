// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Logo correctly 1`] = `
<BrowserRouter>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <Logo
      tab={0}
    >
      <Link
        tabIndex={0}
        title="UXPin logo"
        to="/"
      >
        <LinkAnchor
          href="/"
          navigate={[Function]}
          tabIndex={0}
          title="UXPin logo"
        >
          <a
            href="/"
            onClick={[Function]}
            tabIndex={0}
            title="UXPin logo"
          >
            <SVGInline
              classSuffix="-svg"
              cleanup={Array []}
              cleanupExceptions={Array []}
              component="span"
              svg="test-file-stub"
            >
              <span
                className="SVGInline"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "test-file-stub",
                  }
                }
              />
            </SVGInline>
          </a>
        </LinkAnchor>
      </Link>
    </Logo>
  </Router>
</BrowserRouter>
`;
